lib/locales/ja.yml

```
@@ -119,6 +119,13 @@ ja:
       planet: ["水星", "金星", "地球", "火星", "木星", "土星", "天王星", "海王星"]
       galaxy: ["天の川", "小マゼラン雲", "大マゼラン雲", "アンドロメダ銀河", "Ω星団", "さんかく座銀河", "ケンタウルス座A", "ボーデの銀河", "NGC 253", "M83"]
 
+    subscription:
+      plans: ["Free Trial", "ベーシック", "スターター", "エッセンシャル", "学生", "ブロンズ", "スタンダード", "シルバー", "ゴールド", "プラチナ", "プロフェッショナル", "ビジネス", "ダイヤモンド", "プレミアム"]
```

`"Free Trial"` looks like it can be translated into katakana.

```suggestion
      plans: ["フリートライアル", "ベーシック", "スターター", "エッセンシャル", "学生", "ブロンズ", "スタンダード", "シルバー", "ゴールド", "プラチナ", "プロフェッショナル", "ビジネス", "ダイヤモンド", "プレミアム"]
```

---

Ops...
Thanks for your suggestion. I applied this.